Source/Diagnostics/ReducedDiags/FieldEnergy.H

```diff
@@ -19,16 +19,22 @@ class FieldEnergy : public ReducedDiags
 {
 public:
 
-    /** constructor
-     *  @param[in] rd_name reduced diags names */
+    /**
+     * constructor
+     * @param[in] rd_name reduced diags names
+     */
     FieldEnergy(std::string rd_name);
 
-    /** This function computes the field energy (EF).
-     *  EF = E eps / 2 + B / mu / 2,
-     *  where E is the electric field,
-     *  B is the magnetic field,
-     *  eps is the vacuum permittivity,
-     *  mu is the vacuum permeability. */
+    /**
+     * This function computes the field energy (EF).
+     * EF = E eps / 2 + B / mu / 2,
```

Comment:
Should there be |E|^2 and |B|^2 here?
And maybe also "eps0" and "mu0" instead of "eps" and "mu", just to be 100% correct?